docs/Tutorials/Advanced/ticketing/architecture.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: 'Solution Architecture'
+---
+# Solution Architecture
+In this section, you'll be breaking down the functionality of the ticketing system into project requirements and a solution architecture. This means translating the features from the [ticketing introduction](introduction.md) into tangible goals for the Keypom drop.
+
+## Breaking Down the Problem
+Recall from earlier, the following features are needed:
+
+* No wallet is needed to enter the event or receive a POAP.
+* No wifi is needed at the door.
+* An optional NFT POAP can be minted on-demand for each user that attends the event.
+* Users can optionally onboard onto NEAR if they don't have a wallet.
+
+:::info note
+The NFT POAP is optional to include as the event organizer. You may omit it, or replace it with your own function call if you wish. In this tutorial, the POAP will be minted on the second key use. 
+:::
+
+These can be broken down one by one, starting with the requirement for no wallets. This can be done with any Keypom drop, using the `create_account_and_claim` method. This is available by default, so nothing additional needs to be done to meet this requirement.
```

Comment:
These can be broken down one by one, starting with the requirement for no wallets. This can be done using Keypom's access keys, which once distributed, the attendee can use to create their wallet. 

With traditional events, you would need to burn an NFT in order to gain entry to the event. This model requires the attendee to have a valid wallet with enough $NEAR to cover transaction costs. With a Keypom access key, all you need is a valid private key to gain entrance. 

This key is a simple string and can be turned into a QR code that is able to be scanned.

In order to satisfy the requirement that each ticket must be unique and can only be scanned once, the key can be claimed once it's scanned. To prevent ticket pass-backs, the doorman could check the status of the key and deny entry to the event if the ticket has already been scanned.

*** Last paragraph abt null claims moved to next section re: Keypom Solution. Too specific/nitty gritty for this general solution section

Reply:
The goal of this first section "Breaking down the problem" is just to have overall goals turned into achievable tasks that can be done using Keypom. The next section "Keypom Solution" is more geared towards how the drop can be catered to do those tasks